### PR TITLE
fix: Update supported languages

### PR DESCRIPTION
**Pull request by mrfyda**
depends on codacy/codacy-tools#754

### CODE REVIEW

File: docs/getting-started/supported-languages-and-tools.md [hunk 1]
```diff
@@ -50,9 +50,14 @@ The table below lists all languages that Codacy supports and the corresponding t
   <tbody>
     <tr>
       <td>Apex</td>
+      <!--
       <td><a href="https://pmd.github.io/">PMD</a>,
           <a href="https://semgrep.dev/">Semgrep</a> <a href="#semgrep"><sup>1</sup></a></td>
       <td>-</td>
+      <td><a href="https://semgrep.dev/">Semgrep</a></td>
```
Comment (mrfyda) [inline docs/getting-started/supported-languages-and-tools.md:57]: to uncomment this we need to first add Language.Apex as a supported language to Semgrep in codacy/codacy-tools

File: docs/getting-started/supported-languages-and-tools.md [hunk 2]
```diff
@@ -72,6 +77,9 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td><a href="https://github.com/bridgecrewio/checkov/">Checkov</a></td>
       <td>-</td>
       <td><a href="https://github.com/bridgecrewio/checkov/">Checkov</a>,
+          <!--
+          <a href="https://semgrep.dev/">Semgrep</a> <a href="#yaml-only"><sup>2</sup></a>,
```
Comment (mrfyda) [inline docs/getting-started/supported-languages-and-tools.md:81]: to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools

File: docs/getting-started/supported-languages-and-tools.md [hunk 3]
```diff
@@ -176,15 +188,18 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td>-</td>
       <td>-</td>
     </tr>
+    <!--
```
Comment (mrfyda) [inline docs/getting-started/supported-languages-and-tools.md:191]: to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools

File: docs/getting-started/supported-languages-and-tools.md [hunk 4]
```diff
@@ -212,7 +228,11 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td>Helm</td>
       <td>-</td>
       <td>-</td>
-      <td><a href="https://trivy.dev">Trivy</a> <a href="#yaml-only"><sup>2</sup></a></td>
+      <td>
+          <!--
```
Comment (mrfyda) [inline docs/getting-started/supported-languages-and-tools.md:232]: to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools

File: docs/getting-started/supported-languages-and-tools.md [hunk 5]
```diff
@@ -265,13 +287,21 @@ The table below lists all languages that Codacy supports and the corresponding t
...
       <td>-</td>
-      <td>-</td>
+      <td><a href="https://semgrep.dev/">Semgrep</a></td>
       <td>-</td>
       <td><a href="https://github.com/kucherenko/jscpd">jscpd</a></td>
       <td><a href="https://github.com/detekt/detekt">detekt</a></td>
     </tr>
     <tr>
       <td>Kubernetes</td>
+      <!--
```
Comment (mrfyda) [inline docs/getting-started/supported-languages-and-tools.md:297]: to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools

**Review by codacy-vrhpires — APPROVED** [review]
LGTM

**Pull request merged**